indent/ruby.vim

```diff
@@ -182,7 +187,10 @@ function GetRubyIndent(...)
     let indent = call(function(callback_name), [indent_info])
 
     if indent >= 0
-"      Decho "Match: ".callback_name
+"      Decho "Match: ".callback_name." indent=".indent
```

Comment:
It is, i found the existing debug statements less useful. Figured I'd add some meaningful debug statements commented out like the rest of them.